fiftyone.devicedetection.cloud/examples/cloud/configurator-console/configurator.js

```diff
@@ -34,12 +34,18 @@ Required npm Dependencies:
 - fiftyone.devicedetection
 */
 const path = require('path');
+const require51 = (requestedPackage) => {
+  try {
+    return require(path.join(__dirname, '/../../../node_modules/', requestedPackage));
```

Comment:
I thought here we don't want to specify the explicit path to node_modules - here we just want to have a package name?  it is in the catch clause where we should adjust the path to actual code.